daemon/cluster/convert/swarm.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"strings"
 
+	"net"
```

Comment:
should be 
```
(
    "fmt"
    "strings"
    "net" 
 
    "types "github.com/docker/docker/api/types/swarm"
    ......

Reply:
This is how it shows in my code. dont know why diff looks different 
	"context"
	"fmt"
	"net"
	"strings"
	"time"

Reply:
hrmm... On github it's  definitely showing an extra blank line between "net" and "strings" with "net" coming first and I don't see "time" at all.